example/src/main/java/link/magic/demo/login/MALoginActivity.kt

```diff
@@ -37,6 +38,10 @@ class MALoginActivity : UtilActivity(), AdapterView.OnItemSelectedListener {
         emailButton.setOnClickListener{
             loginWithEmail(it)
         }
+        val recoverAccountButton: Button = findViewById<Button>(R.id.recover_account_btn)
```

Comment:
I think we should put recoverAccount at the bottom of the button list as it's the least likely to be used.